certified-connectors/AzureAD/apiDefinition.swagger.json

```diff
@@ -767,6 +916,24 @@
           "x-ms-summary": "Preferred Language",
           "x-ms-visibility": "advanced"
         },
+        "forceChangePasswordNextSignIn": {
+          "required": [
+            "forceChangePasswordNextSignIn"
+          ],
+          "type": "boolean",
+          "description": "Requires the user to enter a new password",
+          "x-ms-summary": "Rest Password",
+          "x-ms-visibility": "advanced"
+        },
+        "forceChangePasswordNextSignInWithMfa": {
+          "required": [
```

Comment:
"required" as array is used for "object" type and it should be outside of "properties". Please refer to Open Api 2.0 specification

Reply:
done